server/teams.js

```diff
@@ -152,6 +152,12 @@ module.exports = function(imports) {
             return res.status(400).end("You cannot remove the only Admin on your team");
         }
 
+        if (!util.positions.isUserAdmin(req.user)
+            && (user._id.toString() !== req.user._id.toString()))
+        {
+            return res.status(400).end("You cannot remove other users from your team if you are not an Admin");
```

Comment:
`400 Bad Request` is not a good status code to use here. `403 Forbidden` makes more sense.